### ISSUE TITLE
Add CONTRIBUTING.md file

### ISSUE THREAD

**Issue by madiedgar** [legesher/tree-sitter-legesher-python#24]
Add a `CONTRIBUTING.md` file in the root of the repository that contains this content:

```md
See the [Legesher contributing guide](https://github.com/legesher/legesher/blob/master/CONTRIBUTING.md)
```

Simple enough 👍 Please ask to be assigned to this issue so we don't have duplicates 😄

**Comment by zeesh-ali** [comment]
I am picking this

**Comment by madiedgar** [comment]
Awesome @zeesh-ali, I'll assign it to you 😄

**Comment by zeesh-ali** [comment]
Hello Madiedgar, can you please check is the CONTRIBUTING.md file available for your merging.

**Comment by zeesh-ali** [comment]
@madiedgar one check is failling

continuous-integration/appveyor/pr — AppVeyor build failed
can you please guide me, I just started this weekend.

**Comment by zeesh-ali** [comment]
> @madiedgar one check is failling
> 
> continuous-integration/appveyor/pr — AppVeyor build failed
> can you please guide me, I just started this weekend.

The system cannot find the path specified. (os error 3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1

**Comment by rets-mah-ekaj** [comment]
Hello @madiedgar, can I be assigned to this as well?
I have just started open source and I think I can do this.

**Comment by madiedgar** [comment]
Hey @rets-mah-ekaj 👋! You caught me - I forgot to close this issue as it was resolved by @zeesh-ali  in #26, but [this issue](https://github.com/legesher/language-legesher-python/issues/16) is open and I'll be more than happy to help you along the way 👍
